storm-client/src/jvm/org/apache/storm/drpc/LinearDRPCTopologyBuilder.java

```diff
@@ -53,12 +53,12 @@
 // Trident subsumes the functionality provided by this class, so it's deprecated
 @Deprecated
```

Comment:
Not directly related to this PR, but this class has been deprecated for a while. Maybe we should consider removing some of the deprecated classes for 2.0.0?

Reply:
I'll file some separate JIRAs to remove LinerarDRPCTopologyBuilder and TransactionalTopologies that too has been deprecated for a while too.